### DISCUSSION TITLE
Not all walks are alike - 'modes' to focus categories

### DISCUSSION THREAD

**Discussion by salattu** [streetcomplete/StreetComplete#3942] [Ideas]
I like to whip out StreetComplete when I'm walking alone or waiting for a bus. On the one hand I don't like to limit the questions the app presents me, but often I find myself in an area missing building numbers. That information is what I most like to add to OSM.

So I wonder if I could have another way of focusing what questions I see. Right now, I can hide individual items from a long list of items. But what if I want to just focus on one kind of item temporarily? I'm usually open to a variety of questions, but sometimes I only want to see questions of this one type or category.

I might call these focus modes. With just two taps, I want to select between seeing all quests not otherwise disabled (Mode 1) and just a subset (Mode 2), like building data.

**Comment by FloEdelmann** [comment]
You can already create and switch quest presets via the quest selection screen > three-dot menu > "Presets".

**Reply by mnalis** [reply]
I've documented it in [FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#I_want_to_setup_different_groups_of_enabled_quest.2C_and_select_them_depending_on_kind_of_trip_I.27m_taking.3B_is_it_possible.3F) now
